tests/python/gaia-ui-tests/gaiatest/gaia_graphics_test.py

```diff
@@ -48,17 +51,26 @@ def tearDown(self):
         """At the end of test execution, it checks for the errors"""
         self.assertTrue(self.test_passed, msg=self.failcomment)
 
-    def take_screenshot(self):
-        """invokes screen capture event, crops the status bar, and saves to the file"""
+    # marionette hook to override default locale setting
+    def modify_settings(self, settings):
 
-        time.sleep(2)  # compensate for the time taken for actual action on previous step
+        if self.locale != 'undefined':
+            settings['language.current'] = self.locale
+        else:
+            settings['language.current'] = DEFAULT_SETTINGS['language.current']
```

Comment:
Ok, that's the place where it goes wrong.
Suppose I have language.current=nl in my testvars.json file, then I think it should be picked up, but because of this, it picks up the default setting instead of the one I've put in my testvars.json file. Perhaps that else: part can be just removed and then it would be working?